insights/core/remote_resource.py

```diff
@@ -0,0 +1,70 @@
+import requests
+import requests_cache
+
+
+class RemoteResource(object):
+    """
+    RemoteResource class for accessing external Web resources.
+
+    Functions:
+        get(): Calls `get()` function of the RemoteRersource superclass and returns the response payload.
+
+    Each line is parsed into a dictionary with the following keys:
+
+    Examples:
+        >>> from insights.core.remote_resource import RemoteResource
+        >>> rr = RemoteResource()
+        >>> rtn = rr.get("http://google.com")
+        >>> print (rtn.content)
+    """
+
+    timeout = 10
+
```

Comment:
This class attribute needs a docstring, or needs to be documented in the class docstring.

Reply:
Added